chatmd/commands.py

```
@@ -100,20 +111,45 @@ def render_command(options: RenderOptions, env: CommandEnv) -> RenderResult:
             html_path = md_path.with_suffix(".html")
             write_html(doc, html_path, options.html_theme)
 
+        diff_path: Optional[Path] = None
+        if options.diff and not options.dry_run and snapshot is not None:
+            diff_path = write_delta_diff(snapshot, md_path) or None
+            try:
+                snapshot.unlink()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)